### PR TITLE
when creating project add initialize_with_readme option

### PR DESCRIPTION
**Pull request by mustafayildirim**

### CODE REVIEW

File: src/main/java/org/gitlab/api/GitlabAPI.java [hunk 1]
```diff
@@ -1167,7 +1167,8 @@ public GitlabProject createProject(GitlabProject project) throws IOException {
                 .appendIf("request_access_enabled", project.isRequestAccessEnabled())
                 .appendIf("repository_storage", project.getRepositoryStorage())
                 .appendIf("approvals_before_merge", project.getApprovalsBeforeMerge())
-                .appendIf("printing_merge_request_link_enabled", project.isPrintingMergeRequestLinkEnabled());
+                .appendIf("printing_merge_request_link_enabled", project.isPrintingMergeRequestLinkEnabled())
+        	.appendIf("initialize_with_readme",project.isInitializeWithReadme());
```
Comment (timols) [inline src/main/java/org/gitlab/api/GitlabAPI.java]: Please fix the whitespace alignment

File: src/main/java/org/gitlab/api/models/GitlabProject.java [hunk 2]
```diff
@@ -474,8 +477,16 @@ public Boolean isPrintingMergeRequestLinkEnabled() {
     public void setPrintingMergeRequestLinkEnabled(Boolean printingMergeRequestLinkEnabled) {
         this.printingMergeRequestLinkEnabled = printingMergeRequestLinkEnabled;
     }
+    
+    public Boolean isInitializeWithReadme() {
+		return initializeWithReadme;
```
Comment (timols) [inline src/main/java/org/gitlab/api/models/GitlabProject.java]: please fix the whitespace alignment in this file caused by these changes

File: src/main/java/org/gitlab/api/models/GitlabProject.java [hunk 3]
```diff
@@ -497,4 +508,6 @@ public int hashCode() {
         result = 31 * result + (namespace != null ? namespace.hashCode() : 0);
         return result;
     }
+
+
```
Comment (timols) [inline src/main/java/org/gitlab/api/models/GitlabProject.java]: Please remove the extra blank line

**Comment by mustafayildirim** [comment]
Hi @timols,

Sorry for that. I fixed them.

File: src/main/java/org/gitlab/api/models/GitlabProject.java [hunk 4]
```diff
@@ -475,6 +478,14 @@ public void setPrintingMergeRequestLinkEnabled(Boolean printingMergeRequestLinkE
         this.printingMergeRequestLinkEnabled = printingMergeRequestLinkEnabled;
     }
 
+    public Boolean isInitializeWithReadme() {
+	    return initializeWithReadme;
```
Comment (timols) [inline src/main/java/org/gitlab/api/models/GitlabProject.java]: Extra indent here - please fix

File: src/main/java/org/gitlab/api/models/GitlabProject.java [hunk 5]
```diff
@@ -475,6 +478,14 @@ public void setPrintingMergeRequestLinkEnabled(Boolean printingMergeRequestLinkE
...
+    }
+
+    public void setInitializeWithReadme(Boolean initializeWithReadme) {
+		this.initializeWithReadme = initializeWithReadme;
```
Comment (timols) [inline src/main/java/org/gitlab/api/models/GitlabProject.java]: Two extra indents. Can you please fix?

**Pull request merged**